### DISCUSSION TITLE
Skip connections for Graph Attention Network

### DISCUSSION THREAD

**Discussion by PietroNardelli** [danielegrattarola/spektral#290] [Q&A] [answered]
Hi @danielegrattarola,

Thanks for all your great work! I am trying to implement a GAT network for inductive learning (something similar to what is done for the PPI dataset in the original paper) and I was wondering if skip connections are already implemented inside the GATConv layer. If not, where should the skip connections go?

Thanks a lot!

**Comment by danielegrattarola** [comment, marked as answer]
Hi, 

there are no skip connections in the GAT layer, so you'll have to implement them manually. 

Something like:

```python
x, a = inputs
x_new = gat([x, a])
x_out = tf.concat([x_new, x], -1)
```

Cheers
